### ISSUE TITLE
BUG - Accessibility.Test.js (JEST) Automated test fails with new component for qts-v1.2.0

### ISSUE THREAD

**Issue by peteCodes4u** [Quartzion/qts-v1.2.0#47]
**Actual:** JEST tests fail when run due to the implementation of isProd logic for the CWU component.
**Expected** All internal automated tests pass and evaluate all react components.

**Test Steps**
1. Clone the repo - qts-v120-PROD
2. Configure .env file as necessery
3. execute npm install
4. execute npm run build
5. execute npm run develop
6. execute npm run all-tests
     a. observe - JEST test fail due to isProd (import meta) 

 
<img width="898" height="653" alt="Image" src="https://github.com/user-attachments/assets/9ea2bc4f-bc9b-4b3a-b351-fa3f6548b27b" />

**Comment by peteCodes4u** [comment]
This issue has been resolved in the following branch:
[qts-v120-PROD](https://github.com/Quartzion/qts-v1.2.0/tree/qts-v120-PROD)

<img width="838" height="435" alt="Image" src="https://github.com/user-attachments/assets/ab12abae-252c-47e7-a4bf-f583b667b40f" />

**Comment by peteCodes4u** [comment]
<img width="745" height="519" alt="Image" src="https://github.com/user-attachments/assets/5307fab9-d61c-462d-9ff9-1d9af8dbcabf" />
